src/datastore.js

```diff
@@ -418,6 +418,10 @@ s  */
             {
               '@key': 'namespace',
               $: namespaceUrl
+            },
+            {
+              '@key': 'WFSDataStoreFactory:USE_HTTP_CONNECTION_POOLING',
+              $: useHttpConnectionPooling !== false ? 'true' : 'false'
```

Comment:
```suggestion
              $: useHttpConnectionPooling !== false ? true : false
```
I assume the `true` and `false` do not need to be in quotes. 

And I am wondering if this line could be shortened to `!!useHttpConnectionPooling`

Reply:
GeoServer REST API treats the (obviously boolean) value here as string: 

```json
{
  "@key": "WFSDataStoreFactory:USE_HTTP_CONNECTION_POOLING",
  "$": "false"
}
```

In order to avoid some black magic cast I decided to explicitly use this notation.